.github/workflows/ort.yml

```diff
@@ -1,236 +1,260 @@
-              uses: actions/setup-java@v4
-              with:
-                  distribution: "temurin"
-                  java-version: 11
-
-            - name: Cache ORT and Gradle packages
-              uses: actions/cache@v4
-              id: cache-ort
-              with:
-                  path: |
-                      ./ort
-                      ~/.gradle/caches
-                      ~/.gradle/wrapper
-                  key: ${{ runner.os }}-ort
-
-            - name: Checkout ORT Repository
-              if: steps.cache-ort.outputs.cache-hit != 'true'
-              uses: actions/checkout@v4
-              with:
-                  repository: "oss-review-toolkit/ort"
-                  path: "./ort"
-                  ref: "26.0.0"
-                  submodules: recursive
-
-            - name: Install Rust toolchain
-              uses: dtolnay/rust-toolchain@1.78
-
-            - name: Install ORT
-              if: steps.cache-ort.outputs.cache-hit != 'true'
-              working-directory: ./ort/
-              run: |
-                  export JAVA_OPTS="$JAVA_OPTS -Xmx8g"
-                  ./gradlew installDist
-
-            - name: Create ORT config file
-              run: |
-                  mkdir -p ~/.ort/config
-                  cat << EOF > ~/.ort/config/config.yml
-                  ort:
-                    analyzer:
-                      allowDynamicVersions: true
-                      enabledPackageManagers: [Cargo, NPM, PIP, GradleInspector]
-                  EOF
-                  cat ~/.ort/config/config.yml
+          - name: Setup target branch and commit
+            run: |
+              if [ "$EVENT_NAME" == 'workflow_dispatch' ]; then
+                echo "TARGET_BRANCH=$INPUT_TARGET_BRANCH" >> $GITHUB_ENV
+              elif [ "$EVENT_NAME" == 'pull_request' ]; then
+                echo "TARGET_BRANCH=$HEAD_REF" >> $GITHUB_ENV
+              fi
+
+          - name: Checkout target branch
+            uses: actions/checkout@v4
+            with:
+              ref: ${{ env.TARGET_BRANCH }}
+
+          - name: Setup target commit
+            run: |
+              echo "TARGET_COMMIT=`git rev-parse HEAD`" >> $GITHUB_ENV
+
+          - name: Set up JDK 11 for the ORT package
+            uses: actions/setup-java@v4
+            with:
+                distribution: "temurin"
+                java-version: 11
+
+          - name: Cache ORT and Gradle packages
+            uses: actions/cache@v4
+            id: cache-ort
+            with:
+                path: |
+                    ./ort
+                    ~/.gradle/caches
+                    ~/.gradle/wrapper
+                key: ${{ runner.os }}-ort
+
+          - name: Checkout ORT Repository
+            if: steps.cache-ort.outputs.cache-hit != 'true'
+            uses: actions/checkout@v4
+            with:
+                repository: "oss-review-toolkit/ort"
+                path: "./ort"
+                ref: "26.0.0"
```

Comment:
Can you verify that the version is relevant? I think it's for legacy reasons, and it should be removed.

Reply:
This is unrelated to the change. We will update separately, i dont want to deal with regressions now 
https://github.com/valkey-io/valkey-glide/issues/2709